### DISCUSSION TITLE
Some interesting free samplesets for GrandOrgue

### DISCUSSION THREAD

**Discussion by larspalo** [GrandOrgue/grandorgue#242] [General]
Originally posted 2018-10-16 11:01:51.869000 by briansweetnam in GrandOrgue discussion forum on SourceForge<br><br>Hi, I recently discovered this website (in France) with some really interesting samplesets.  There are some really nice instruments that have not previously been available for GrandOrgue.  Take a look and decide for yourself.

There are over 100 different sampleset of instruments like barrel organs, electronic organs, pipe organs, theatre organs, piano&#039;s, harpsichords, trumpets, and so on. Here is the link to the website:
https://sites.google.com/site/orguesdecinema/

**Comment by larspalo** [comment]
Originally posted 2019-01-29 15:02:17.040000 by user1pl in GrandOrgue discussion forum on SourceForge<br><br>Very interesting! Especially samplesets with trumpets and bells. I can confirm it works. The link to that site really deserves to be included in the &quot;Samplesets&quot; page.

**Comment by larspalo** [comment]
Originally posted 2019-01-29 17:50:39.981000 by tweedledee in GrandOrgue discussion forum on SourceForge<br><br>Nice find.  Thanks.

**Comment by larspalo** [comment]
Originally posted 2019-01-29 18:00:00.424000 by briansweetnam in GrandOrgue discussion forum on SourceForge<br><br>I agree with you user1PL!

**Reply by Venturedog1** [reply]
Hello. It seems the link is now dead. May someone please post a new link if available? Thank you.

**Reply by hnb2907** [reply]
I'm guessing, could it be this one with a different URL?  https://odfgrandorguefr.wixsite.com/home

Thanks for bumping the thread, I'm having a look at the Hammond B3 sampleset, which I was looking for ages ago :)

Cheers,
Chris

**Reply by Venturedog1** [reply]
Chris, THANK YOU for the link. Great resource!

**Reply by hnb2907** [reply]
No problem!  You did me a favour, I was looking for the same site a couple of years ago, but didn't find it.  So you reminded me to look again! :)

C.

**Comment by larspalo** [comment]
@Venturedog1 & @hnb2907  Actually not intending to dampen the spirit of excitement of anyone, but this site, and the name Lacaud brings up memories of a person that obviously used other peoples material without giving proper recognition of it == committing fraud. See https://sourceforge.net/p/ourorgan/mailman/message/31647660/ and https://sourceforge.net/p/ourorgan/mailman/ourorgan-developers/thread/1384681975.14233.YahooMailNeo%40web171206.mail.ir2.yahoo.com/#msg31647660 for the whole thread.

It might be that some improvements have been done over the years though. I don't have any real interest in investigating the sample sets more than I've already did about 12 years ago, but that history is one reason that this site likely won't be included in the wiki for sample sets of GrandOrgue. Just letting you know...

Of course, feel free to use whatever sample sets you find and like with GrandOrgue for you own personal pleasure!
